### ISSUE TITLE
Ambiguous description to the input of CrossEntropyLoss

### ISSUE THREAD

**Issue by AMR-KELEG** [pytorch/pytorch#15283]
The cross entropy loss requires a linear output layer with no softmax activation function.
However, the documentation isn't specifying this in a clear way IMHO.
https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py#L824
Should it be rephrased to something like:
    ```The `input` is expected to contain scores for each class. Don't normalize these scores into probabilities.```

**Comment by vishwakftw** [comment]
@soumith This can be closed; the fix has landed in https://github.com/pytorch/pytorch/commit/492b7d410b03fef9b0438f64fb5d4383402b973e
